packages/components/descriptions/src/descriptions-cell.ts

```diff
@@ -94,13 +94,15 @@ export default defineComponent({
             colSpan: span,
           },
           [
-            h(
-              'span',
-              {
-                class: [ns.e('label'), labelClassName],
-              },
-              label
-            ),
+            label != null
```

Comment:
We can utilize `_.isNil` in this.

```ts
import { isNil } from 'lodash-unified'
```

```suggestion
            isNil(label)
```

Reply:
ok. updated 👀